Resources/Prototypes/Entities/Clothing/Head/hats.yml

```diff
@@ -347,6 +347,9 @@
     sprite: Clothing/Head/Hats/plaguedoctor.rsi
   - type: Clothing
     sprite: Clothing/Head/Hats/plaguedoctor.rsi
+  - type: Tag
+    tags:
+    - HidesHair
```

Comment:
are you sure you not losing any other tags from the parent?

Reply:
Thanks for help. 
Well... I'm loosing some needed tags...
I'll fix that as soon as possible